sensor_msgs/JointState initialState
sensor_msgs/JointState goalState
---
trajectory_msgs/JointTrajectory trajectory
